ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -506,6 +506,9 @@ literals:
 - phase3_powerfactor_sensor
 - powerfactor_sensor
 - reactive_power_sensor
+- phase1_reactive_power_accumulator
```

Comment:
Please confirm if this is actually an `accumulator` or if it should be a `sensor`

Reply:
Checked and updated to sensor.

Reply:
Updated to sensor